spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
```

Comment:
You can add https://github.com/rubocop/rubocop-rspec to help you format your test files correctly